### DISCUSSION TITLE
Running Terria without a server

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [TerriaJS/terriajs#5088] [General] [answered]
I remember some demo (I think from @stevage?) years ago running Terria from a static bucket or URL (e.g. without a server). 

I'm hoping I just don't have my full Google Fu this morning -- if someone could point me in the right direction I'd definitely appreciate it!

**Comment by hardreddata** [comment, marked as answer]
@steve9164 prepared a branch for this recently. 

See https://github.com/TerriaJS/TerriaMap/compare/static-map

**Reply by tephenavies** [reply]
Thanks. I forgot about this. I'll write this into docs in the new year.
